app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
+  </dl>
+  <hr />
+  <%= render partial: 'shared/searchworks_item_information', locals: { request: request } %>
```

Comment:
Yeah, Spotlight is where I have had trouble with it.  I know it works as we've implemented it but I found some rough edges working with it.

I'm working on implementing a `current_request` pattern.